release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -230,17 +240,27 @@ private void onTextChange(string newText)
 			return;
 		
 		this.objUnityInput.text = newText;
-		if (this.objUnityInput.onValueChanged != null) this.objUnityInput.onValueChanged.Invoke(newText);
+		if (this.objUnityInput.onValueChanged != null)
+			this.objUnityInput.onValueChanged.Invoke(newText);
 	}
 
 	private void onTextEditEnd(string newText)
 	{
 		this.objUnityInput.text = newText;
-		if (this.objUnityInput.onEndEdit != null) this.objUnityInput.onEndEdit.Invoke(newText);
+		if (this.objUnityInput.onEndEdit != null)
+			this.objUnityInput.onEndEdit.Invoke(newText);
 	}
 
 	public override void OnPluginMsgDirect(JsonObject jsonMsg)
 	{
+		if(this.gameObject.activeInHierarchy)
+			StartCoroutine(PluginsMessageRoutine(jsonMsg));
```

Comment:
Should we handle the message even if the GameObject is disabled. I was thinking of doing something like `PluginMsgHandler.getInst().gameObject.StartCoroutine(PluginsMessageRoutine(jsonMsg));` here, because the `PluginMsgHandler` singleton should always exist. For example if we get a text update message from the native side now it can be lost if this gameobject is disabled, right? I don't know if that is very likely, but just in case.